### DISCUSSION TITLE
posts/zerotier-ssh/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#11] [Announcements]
# posts/zerotier-ssh/

一些内网的主机可以访问公网，但是关闭了公网SSH端口。可以通过ZeroTier搭建VPC，让内网主机与client位于同一虚拟局域网中，实现SSH连接。

https://www.haoyep.com/posts/zerotier-ssh/

**Comment by uchouT** [comment]
博主你好，我用这个方法想访问内网主机的WSL，但是总是会连接超时，我已经通过[这个方法](https://www.sxrhhh.top/blog/2023/12/14/connect-to-wsl/)开启了外部访问，ssh的配置上应该也没有问题 (使用telnet可以连接)。可以求解一下吗，谢谢。

**Reply by leegical** [reply]
可能是zerotier被墙了/服务不可用，先排查一下在正常机器上能否使用
